src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -178,7 +176,7 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics, isAnonimit
                         #{convertString(item.name, ' ', '')}
                       </Text>
                       {index !== topicSuggestion.length - 1 && (
-                        <View style={{height: 1, marginTop: 5, backgroundColor: COLORS.gray1}} />
+                        <View style={{height: 1, marginTop: 5, backgroundColor: '#C4C4C4'}} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, avoid hardcoding color values.

```diff
- <View style={{height: 1, marginTop: 5, backgroundColor: '#C4C4C4'}} />
+ <View style={{height: 1, marginTop: 5, backgroundColor: colors.gray1}} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->